pkg/services/alerting/notifiers/discord.go

```diff
@@ -213,3 +215,11 @@ func (dn *DiscordNotifier) embedImage(cmd *models.SendWebhookSync, imagePath str
 
 	return nil
 }
+
+func notEmpty(metric string) string {
+	if metric == "" {
+		return "Empty metric"
```

Comment:
I think we should change the message to something like `<NO_METRIC_NAME>` or `<INVALID_METRIC_NAME>` rather than anything else, since the way you have it right now, appears like:

![image](https://user-images.githubusercontent.com/15115078/109208826-7d4d4780-77b3-11eb-81fd-208601215d7b.png)

which basically feels like there's a metric called empty!

Reply:
Totally get that. `<NO_METRIC_NAME>` it is, because technically `{{hostname}}` is valid, it just doesn't show up on the server.